libs/injection/bbhnet/injection/injection.py

```diff
@@ -92,12 +136,16 @@ def generate_gw(
         psi = p["psi"]
 
         # Generate signal in IFO
-        polarizations = waveform_generator.time_domain_strain(p)
         signal = np.zeros(waveform_size)
         for mode, polarization in polarizations.items():
             # Get ifo response
             response = ifo.antenna_response(ra, dec, geocent_time, psi, mode)
-            signal += response * sig.filtfilt(b, a, polarization)
+            signal += (
```

Comment:
this is kind of a kooky way for black to factor this. Is there a reason it can't be `signal += response * polarization[i]` all on one line?

Reply:
Pretty sure that's a result of one the pre-commit hooks editing things. I don't recall writing it out that way...

Reply:
It's all good I had to add another commit in anyway to fix a different issue with Black so I just fixed this quickly